src/schema/sale/__tests__/index.test.js

```diff
@@ -420,99 +420,102 @@ describe("Sale type", () => {
           live_start_at: moment().subtract(1, "days"),
           registration_ends_at: moment().subtract(2, "days"),
         },
-        "In Progress",
+        "in progress",
       ],
+      [{ end_at: moment().subtract(1, "days") }, null],
       [
         {
           auction_state: "open",
           live_start_at: moment().subtract(2, "days"),
           registration_ends_at: moment().subtract(3, "days"),
         },
-        "In Progress",
+        "in progress",
       ],
       [
         {
           live_start_at: moment().add(10, "minutes"),
           registration_ends_at: moment().subtract(2, "days"),
         },
-        "Live in 10 minutes",
+        "live in 10 minutes",
```

Comment:
one thing I'm thinking to add an arg for is `useAbbreviations` or something like that.

In the artwork brick context, we might prefer `live in 10m`, `live in 2hr`, etc.
But in the slightly more prominent labeling this is currently being used in, we might want to keep then hours, minutes, etc.

That can come later as it's an enhancement to the new usage, and that feels like a reasonable arg for this field if we do in fact need it.